fiftyone/core/stages.py

```diff
@@ -4092,7 +4092,7 @@ def _params(cls):
         return [
             {
                 "name": "slices",
-                "type": "NoneType|list<str>|str",
+                "type": "NoneType|str|list<str>",
```

Comment:
There are a handful of other instances of `list<str>|str` in other view stages, which I thought was required so that `a,b` is parsed as `["a", "b"]` rather than being treated as a string.

Isn't there another way to address this issue that preserves the possibility of providing a single string or `list,of,string`?

Reply:
Yes, fixed